.gitignore

```diff
@@ -61,4 +61,7 @@ coverage.xml
 *.py,cover
 .hypothesis/
 .pytest_cache/
-cover/
\ No newline at end of file
+cover/
+
+# virtual environment
+gangaenv
```

Comment:
why is this here?

Reply:
Oh! This wasn't supposed to go in. Will fix it.